apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/BaseNode/BaseNode.tsx

```diff
@@ -40,8 +40,12 @@ const connectionNodeIdSelector = (state): string | null =>
 
 interface BaseNodeProps {
   id?: string
-  targetHandle?: 'show' | 'hide' | 'disabled'
-  sourceHandle?: 'show' | 'hide' | 'disabled'
+  targetHandle?:
+    | 'none' // Handle is not rendered - Default value
+    | 'show' // Handle is rendered and visible
+    | 'hide' // Handle is rendered but not visible. (Used for instances like the social preview node to conditionally display the handle)
+    | 'disabled' // Handle is rendered and visible, but not connectable
```

Comment:
Do you also think we should also do this for the source handle? Could worth making this into a ENUM at this point
```suggestion
    | 'none' // not rendered - Default value
    | 'show' // rendered, visible and connectable
    | 'hide' // rendered, not visible or connectable
    | 'disabled' // rendered, visible, not connectable
```